### ISSUE TITLE
Glass shards to damage to the user when attacked with

### ISSUE THREAD

**Issue by woothie** [HeavensGate/NSS-Eternal#31]
Ripped right outta code priorities:
Glass shards damage their user's hands slightly. Not sure if we want to add sharp to this. Not sure if we want this to be enough to make them drop things.I'm for adding sharp. Just remember, stabbing with a glass shard 5-15 times will result in you not being able to use that hand until you get medical attention, and you could thereby bleed out from it. Which will surprise people thinking (foolishly) that we still hand out generic.sharp from windows. Also, note that shivs specifically have a handguard to prevent this problem. So if they pull in glass_shard code, leave that bit out. :p (codernote: this damage is on use on another mob or object as an attack. Throwing glass shards or turning them into other weapons is the ideal situation. Carrying them around is harmless. For now, we'll say that wearing non-latex gloves prevents the damage, but latex gloves are ruined if used to make a glass shard attack.

**Comment by Davnik** [comment]
Note that "I'm for adding sharp." was static's comment, and all other was my own. Yes, this is a series of ideas. Bite me.

~Glass shards do some damage to your active hand when used to attack a mob, about 2-5 brute sharp non-edge. We don't want this to be debilitating until after 5-15 stabs or so, which we can tailor around for balance later.
~Wearing gloves changes the process. Most gloves prevent it entirely. Fingerless gloves proc a 50% chance of damaging your hands. Latex gloves are ruined and provide no protection, as well as not providing a barrier against fingerprints or infection. In effect, we might want to delete the latex gloves with a message sent explaining that they have become useless.
~Carrying glass shards will not harm you. Getting a good enough grip to stab someone with one will harm you.
~Shivs ignore this property, as they specifically have a handguard to prevent this issue.

**Comment by staticcharged** [comment]
True story, I really am all for adding sharp.

**Comment by Davnik** [comment]
This is not a bug, this is an enhancement only. We want to add a feature that causes glass shards to damage their users

**Comment by Jacquar** [comment]
Yes. This. Can shivs be made using shards and another item? If not, can we make it be duct tape? That tape needs more uses. Especially for antags.
